.github/workflows/authorization-ci.yml

```
@@ -0,0 +1,57 @@
+env:
+  IMAGE_NAME: decidish/auth-service
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
```

actionlint (1.7.9)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)